proposals/003-installer-guidelines.md

```diff
@@ -0,0 +1,102 @@
+## Introduction
+
+The Haskell Foundation would like see a central Haskell download page in which the Haskell curious
+could easily download and run a installer to provide a standard Haskell environment that supports
+the principal build tools (`cabal-install` and `stack`) on the main platforms (Linux, macOS and
+Windows). All Haskell introductory material (text books, blog posts, etc.) should be able to point
+to the Haskell Foundation download page and work on the assumption that it will yield their
+widely-supported Haskell development environment.
+
+To this end this proposal covers two things:
+
+  * To publish a set of minimal requirements for a Haskell installer that we are prepared to
+    recommend along with a roadmap of the minimal requirements that we are looking to add in the
+    future. (The initial requirements will not require precompiled libraries or editor/IDE
+    environments but we would like to move in that direction, for example.)
+
+  * To provide a prototype Windows installer for cabal-install-based Haskell development
+    environments based on the stack codebase. This is the coverage we are missing at the moment and
```

Comment:
Strong 👎 

The proposal should not mention any existing installer tools. It should be neutral and not care who or what satisfies these requirements.

There has been far too much political play in the development of this proposal and we need to remove all of that from it.